Detectors/CPV/calib/include/CPVCalibration/PedestalCalibrator.h

```diff
@@ -87,14 +87,14 @@ class PedestalCalibrator final : public o2::calibration::TimeSlotCalibration<o2:
   std::vector<o2::ccdb::CcdbObjectInfo> getCcdbInfoEfficienciesVector() { return mCcdbInfoPedEfficienciesVec; }
   std::vector<std::vector<float>> getEfficienciesVector() { return mPedEfficienciesVec; }
```

Comment:
this returns a copy of the vectors. Is this what you want? Usually it is enough to do e.g.
```
const std::vector<o2::ccdb::CcdbObjectInfo>&  getCcdbInfoEfficienciesVector() const { return mCcdbInfoPedEfficienciesVec; }
```

Reply:
Let me try to use const std::vector<o2::ccdb::CcdbObjectInfo>&

Reply:
ok, then you can do 
```
std::vector<o2::ccdb::CcdbObjectInfo>&  getCcdbInfoEfficienciesVector() { return mCcdbInfoPedEfficienciesVec; }
```
my point was why do you need return a copy.

Reply:
Hi @shahor02 
yes, I implemented return values as references. CCDBinfo as normal references and CCDB objects as const references. Just for my understanding, what is approximate time for alien token to be imlemented into fullCI?

Reply:
Hi @sevdokim, unfortunately, I don't know, it is followed here: https://alice.its.cern.ch/jira/browse/O2-2760